tests/90jira/SYN-391.pl

```diff
@@ -0,0 +1,30 @@
+multi_test "Setting push rules twice 500s  SYN-391",
+    requires => [qw( register_new_user http_clients do_request_json_for )],
+    do => sub {
+        my ( $register_new_user, $clients, $do_request_json_for ) = @_;
+        my $http = $clients->[0];
+        my $alice;
+
+        Future->needs_all(
+            $register_new_user->($http, "90jira-SYN-391_alice"),
+        )->then( sub {
+            ($alice) = @_;
```

Comment:
Wants whitespace inside parens `( $vars, here.... )`
Wants blank line after the `@_`-unpacking line.